test/scenarios/sflight/read.test.js

```diff
@@ -44,6 +44,7 @@ describe('SFlight - Read', () => {
     // REVISIT: works in sflight not in tests
     // 'Bookings?$apply=concat(groupby((BookingID,ConnectionID,CurrencyCode_code,FlightDate,ID,TravelID,airline,status))/aggregate($count%20as%20UI5__leaves),aggregate(FlightPrice,CurrencyCode_code),groupby((airline,airlineName),aggregate(FlightPrice,CurrencyCode_code))/concat(aggregate($count%20as%20UI5__count),top(53)))',
     'Bookings?$apply=groupby((airline,airlineName),aggregate(FlightPrice with average as avgPrice,FlightPrice with max as maxPrice,FlightPrice with min as minPrice))&$skip=0&$top=1',
+    `Bookings?$apply=filter(airline eq 'EA' and status eq 'B')/groupby((ID),aggregate(FlightPrice,CurrencyCode_code))&$count=true&$skip=0&$top=1`
```

Comment:
we don't really test that the count is correct.
The request went throught before as well but included a wrong count

Reply:
Also having a more simplified query that reproduces this issue would make it easier to understand the problem later when someone breaks the test.